lib/xcknife/test_dumper.rb

```diff
@@ -294,6 +297,32 @@ def simctl
       @simctl_path
     end
 
+    def wrapped_simctl(args)
+      args = [simctl] + args
+
+      unless gtimeout.nil?
+        args.prepend(gtimeout).flatten
+      end
+
+      args
+    end
+
+    def gtimeout
+      return nil unless @simctl_timeout > 0
```

Comment:
NIT: If this returned an empty array instead of nil, the `wrapped_simctl` would be a bit simpler of: `args = [*gtimeout, simctl] + args`